test/companion/companion_web/companion_web_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule Companion.CompanionWebTest do
+  use Companion.DataCase
+
+  alias Companion.CompanionWeb
+
+  describe "applications" do
+    alias Companion.CompanionWeb.Application
+
+    @valid_attrs %{name: "some name", token: "7488a646-e31f-11e4-aace-600308960662"}
+    @update_attrs %{name: "some updated name", token: "7488a646-e31f-11e4-aace-600308960668"}
+    @invalid_attrs %{name: nil, token: nil}
+
+    def application_fixture(attrs \\ %{}) do
+      {:ok, application} =
+        attrs
+        |> Enum.into(@valid_attrs)
+        |> CompanionWeb.create_application()
+
+      application
```

Comment:
I often find myself writing a little `defp assume_ok({:ok, value}), do: value` function to pipe through instead of extracting the value with a pattern match.